tower-steer/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
```

Comment:
We probably want to tidy this up a bit and make it more like the `Cargo.toml` of the various other `tower-` subcrates. A simple `README.md` would also be good to add.